typescript/nuxtjs-prisma-nuxt-module/README.md

```diff
@@ -0,0 +1,36 @@
+# Nuxt Example App Using Prisma Nuxt Module
+
+This example app showcases the usage of the [Prisma Nuxt module](https://github.com/prisma/nuxt-prisma).
+
```

Comment:
Awkward jump from the intro sentence to this list. Why does this list exist? What are these elements?